docs/sender_adaptors.adoc

```diff
@@ -175,7 +175,16 @@ xref:schedulers.adoc#_inline_scheduler[`inline_scheduler`].
 Found in the header: `async/retry.hpp`
 
 `retry_until` works like `retry`, but takes a predicate. If the predicate
-returns false, `retry_until` can complete on the error channel.
+returns true, `retry_until` can complete on the error channel.
```

Comment:
question: I'm not sure what this means. I assume `retry_until` will also complete with `set_error` and propagate the error?

Reply:
Resolving comment so I can merge  (o;